native_client/java/standalone.md

```diff
@@ -0,0 +1,193 @@
+# DeepSpeech for standalone Java
+
+### Adding DeepSpeech to your Java project
+>Note: You can do this on your own way too, but this works with the example usage code provided below.
+1. Copy `libdeepspeech.jar` from `./build/` into `{YOUR PROJECT ROOT}/libs/`
+2. Copy '*.so' from `./build/` into `{YOUR PROJECT ROOT}/src/main/resources/jni/x86_64/`
+3. Modify your `build.gradle` file to include:
+```groovy
+plugins {
+   	id 'com.github.johnrengelman.shadow' version '5.0.0' 
+}
+
+repositories {
+    flatDir {
+			dirs 'libs'
+    }
+}
+
+dependencies {
+    compile ':libdeepspeech'
+}
+```
+4. You can now compile your project with `./gradlew shadowJar`.

```

Comment:
what's shadow far ?

Reply:
shadowJar is a way of packaging your dependencies in with your jar. If you just do ./gradlew build, you'd get your jar but libdeepspeech would not be included